metagpt/logs.py

```diff
@@ -31,7 +35,8 @@ def define_log_level(print_level="INFO", logfile_level="DEBUG", name: str = None
 
 
 def log_llm_stream(msg):
-    _llm_stream_log(msg)
+  if _print_level in ["DEBUG"]:
+        _llm_stream_log(msg)
```

Comment:
It's recommended to add this condition inside the `_llm_stream_log` function. Users can utilize `set_llm_stream_logfunc` to customize the behavior of their own `llm_stream_log` , but they may not want it to be affected by the `log_level` flag. You can define the function like this:

```python

def _llm_stream_log(msg):
    if _print_level in ["DEBUG"]:
        print(msg, end="")
```

Reply:
reasonable consideration！ change is made accordingly